input_parser.star

```diff
@@ -134,12 +137,22 @@ DEFAULT_L1_ARGS = {
     # "minimal" preset will spin up a network with minimal preset. This is useful for rapid testing and development.
     # 192 seconds to get to finalized epoch vs 1536 seconds with mainnet defaults
     # Please note that minimal preset requires alternative client images.
-    "l1_preset": "minimal",
+    "l1_preset": "mainnet",
     # Number of seconds per slot on the Beacon chain
     # Default: 12
     "l1_seconds_per_slot": 1,
-    # The amount of ETH sent to the admin, sequence, aggregator and sequencer addresses.
-    "l1_funding_amount": "100ether",
+    # The amount of ETH sent to the admin, sequence, aggregator, sequencer and other chosen addresses.
+    "l1_funding_amount": "1000000ether",
+    # Default: 2
+    "l1_participants_count": 1,
+    # Default: 2048
+    "l1_eth1_follow_distance": 1,
```

Comment:
[ETH1_FOLLOW_DISTANCE
](https://eth2book.info/capella/part3/config/configuration/#eth1_follow_distance)This is used to calculate the minimum depth of block on the Ethereum 1 chain that can be considered by the Eth2 chain: it applies to the [Genesis](https://eth2book.info/capella/part3/initialise/#initialise-state) process and the [processing of deposits](https://github.com/ethereum/consensus-specs/blob/v1.3.0/specs/phase0/validator.md#process-deposit) by validators. The Eth1 chain depth is estimated by multiplying this value by the target average Eth1 block time, [SECONDS_PER_ETH1_BLOCK](https://eth2book.info/capella/part3/config/configuration/#seconds_per_eth1_block).

The value of ETH1_FOLLOW_DISTANCE is not based on the expected depth of any reorgs of the Eth1 chain, which are rarely if ever more than 2-3 blocks deep. It is about providing time to respond to an incident on the Eth1 chain such as a consensus failure between clients.

This parameter [was increased](https://github.com/ethereum/consensus-specs/pull/2093/files) from 1024 to 2048 blocks for the beacon chain mainnet, to allow devs more time to respond if there were any trouble on the Eth1 chain.

The whole follow distance concept has been made redundant by the Merge and may be removed in a future upgrade, so that validators can make deposits and become active more-or-less instantly.

Reply:
Can you add those comments directly in the code? This PR will be forgotten ^^